os/kernel/debug/sysdbg.c

```diff
@@ -421,53 +428,69 @@ static ssize_t sysdbg_write(FAR struct file *filep, FAR const char *buffer, size
 
 	sscanf(buffer, "%s %s", arg1, arg2);
 
-	if (strncmp("enable_monitor", arg1, 14) == 0) {
+	if (strncmp("help", arg1, 5) == 0) {
+		show_usage();
+		return 0;
+	}
+
+	switch (atoi(arg1)) {
+	case 1:		/*Enable monitor*/
```

Comment:
How about using define values for many cases?

Reply:
Ok will update that.